src/Nethermind/Nethermind.Evm/Precompiles/IPrecompile.cs

```diff
@@ -13,9 +13,9 @@ public interface IPrecompile
 
         long BaseGasCost(IReleaseSpec releaseSpec);
 
-        long DataGasCost(in ReadOnlyMemory<byte> inputData, IReleaseSpec releaseSpec);
+        long DataGasCost(ReadOnlyMemory<byte> inputData, IReleaseSpec releaseSpec);
 
-        (ReadOnlyMemory<byte>, bool) Run(in ReadOnlyMemory<byte> inputData, IReleaseSpec releaseSpec);
```

Comment:
do we have to remove `in`?

Reply:
I removed it because you can't access a ref type in a lambda function like in the multimul precompiles. Also `ReadOnlyMemory` is a small type intended to be passed by value so I don't see any advantage to adding indirection (https://stackoverflow.com/a/49297998)